Module not found issue

After doing all the said steps and installing dependencies, it's showing this

sayantan@sayantan:~/Documents/Node Apps/roachApp/test/example-app-node-sequelize$ node app.js 
internal/modules/cjs/loader.js:883
  throw err;
  ^

Error: Cannot find module './utils/logger'
Require stack:
- /home/sayantan/Documents/Node Apps/roachApp/node_modules/sequelize/lib/data-types.js
- /home/sayantan/Documents/Node Apps/roachApp/node_modules/sequelize/lib/utils.js
- /home/sayantan/Documents/Node Apps/roachApp/node_modules/sequelize/lib/dialects/postgres/query-generator.js
- /home/sayantan/Documents/Node Apps/roachApp/node_modules/sequelize-cockroachdb/source/index.js
- /home/sayantan/Documents/Node Apps/roachApp/test/example-app-node-sequelize/app.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
    at Function.Module._load (internal/modules/cjs/loader.js:725:27)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/home/sayantan/Documents/Node Apps/roachApp/node_modules/sequelize/lib/data-types.js:10:20)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/sayantan/Documents/Node Apps/roachApp/node_modules/sequelize/lib/data-types.js',
    '/home/sayantan/Documents/Node Apps/roachApp/node_modules/sequelize/lib/utils.js',
    '/home/sayantan/Documents/Node Apps/roachApp/node_modules/sequelize/lib/dialects/postgres/query-generator.js',
    '/home/sayantan/Documents/Node Apps/roachApp/node_modules/sequelize-cockroachdb/source/index.js',
    '/home/sayantan/Documents/Node Apps/roachApp/test/example-app-node-sequelize/app.js'
  ]
}

--- a/.gitignore
+++ b/.gitignore
@@ -1,0 +1,1 @@
+node_modules/

--- a/README.md
+++ b/README.md
@@ -19,6 +19,6 @@ To run the code:
 1. In your terminal, from the top of this project directory:
 
    ```shell
-   $ npm install sequelize sequelize-cockroachdb
+   $ npm install
    $ node app.js
    ```

--- a/app.js
+++ b/app.js
@@ -1,25 +1,8 @@
 const Sequelize = require("sequelize-cockroachdb");
-// For secure connection:
-//const fs = require('fs');
 
 // Connect to CockroachDB through Sequelize.
-var sequelize = new Sequelize({
-  dialect: "postgres",
-  username: "max",
-  password: "roach",
-  host: "localhost",
-  port: 26257,
-  database: "bank",
-  dialectOptions: {
-    ssl: {
-      rejectUnauthorized: false,
-      // For secure connection:
-      /*ca: fs.readFileSync('certs/ca.crt')
-                .toString()*/
-    },
-  },
-  logging: false,
-});
+const connectionString = process.env.DATABASE_URL
+const sequelize = new Sequelize(connectionString)
 
 // Define the Account model for the "accounts" table.
 const Account = sequelize.define("accounts", {

--- a/package.json
+++ b/package.json
@@ -1,0 +1,6 @@
+{
+  "dependencies": {
+    "sequelize": "^6.17.0",
+    "sequelize-cockroachdb": "^6.0.5"
+  }
+}

